dspace/config/launcher.xml

```diff
@@ -373,4 +373,11 @@
             <class>org.dspace.app.util.InitializeEntities</class>
         </step>
     </command>
+    <command>
+        <name>anonymize-statistics</name>
+        <description>Anonymize the ip values of the solr statistics</description>
+        <step>
+            <class>com.atmire.dspace.statistics.AnonymizeStatistics</class>
```

Comment:
This looks like a typo to me. Shouldn't that be `org.dspace.statistics.AnonymizeStatistics`?

Reply:
Yes this is a typo, this needs to be fixed.

Reply:
You are correct, fixed this!